py3status/modules/external_script.py

```diff
@@ -82,10 +92,26 @@ def external_script(self):
         else:
             output_text = ''
 
+        self.script_data = {
+            'line': len(output_lines),
+            'output': output_text,
+            'output_full': ' '.join(output.splitlines())
+        }
         response['full_text'] = self.py3.safe_format(
-            self.format, {'output': output_text})
+            self.format, self.script_data
+        )
+        self.script_data['output_full'] = output
```

Comment:
@lasers why is this line needed? it already contains full output since line 98, isn't?

Reply:
To remove newlines, put them all on one same line for the bar.
And to preserve newlines for the notification.

Try `cal`.